src/js/pages/Setup.js

```diff
@@ -462,12 +466,67 @@ class Setup extends mixin(StoreMixin) {
           validationErrorText: this.getErrors('ip_detect_script'),
           value: this.state.formData.ip_detect_script
         }
-      ]
+      ],
+      {
+        fieldType: 'checkboxMultiple',
+        name: 'nested_cluster_options',
+        formRowType: 'inline',
+        value: [
+          {
+            name: 'telemetry_enabled',
+            label: (
+              <span>
+                Send Anonymous Telemetry
+                <Tooltip content={
+                    <span>
+                      Enter or upload a script that runs on each node in the
+                      cluster and outputs the node’s local IP address. <a
+                        href={`${Config.documentationURI}/advanced-installer/create-a-script-for-ip-address-discovery/`}
+                        target="_blank">
+                        Learn more
+                      </a>.
+                    </span>
+                  }
+                  width={300} wrapText={true} />
+              </span>
+            ),
+            checked: this.state.formData.telemetry_enabled
+          },
+          {
+            name: 'oauth_enabled',
+            label: (
+              <span>
+                Enable Authentication
+                <Tooltip content={
+                    <span>
+                      Enter or upload a script that runs on each node in the
+                      cluster and outputs the node’s local IP address. <a
+                        href={`${Config.documentationURI}/advanced-installer/create-a-script-for-ip-address-discovery/`}
+                        target="_blank">
+                        Learn more
+                      </a>.
+                    </span>
+                  }
+                  width={300} wrapText={true} />
+              </span>
+            ),
+            checked: this.state.formData.oauth_enabled
+          }
+        ]
+      }
     ];
 
     return Hooks.applyFilter('FormDefinition', formDefintion);
   }
 
+  getFormRowClass(definition) {
```

Comment:
I implemented `classnames` inside this method, but we still need the method. This is the only way to conditionally set a form row's class depending on its contents.